test.js

```diff
@@ -449,6 +449,71 @@ test('should register under-pressure with underPressure: true - and expose route
   })
 })
 
+test('should decorate with next render function', t => {
+  t.plan(3)
+
+  const fastify = Fastify()
+  t.tearDown(() => fastify.close())
+
+  fastify
+    .register(require('./index'))
+    .after(() => {
```

Comment:
I would recommend removing this `.after()` and just do an `await fastify.register(..)`

Reply:
I just copied the other test. happy to use async-await instead, tho

Reply:
hmm, I end up getting 503 errors when running the tests then... I can push what I did, might have done something dumb

Reply:
[`674fd44` (#201)](https://github.com/fastify/fastify-nextjs/pull/201/commits/674fd4490597230bd14300f9e58efb8ce1d0329e)